src/platform/cmx/include/cmsis/core_cmFunc.h

```diff
@@ -0,0 +1,87 @@
+/**************************************************************************//**
```

Comment:
for all of the `cmsis` stuff, should we use a `git submodule` for it?  Is it already on github somewhere?